schemas/common/accesscontrolpolicy.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/accesscontrolpolicy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Access Control Policy",
+  "description": "Schema for the Access Control Policy of a resource",
+  "type": "object",
+  "definitions": {
+    "accesscontrolpolicy": {
+      "type": "object",
+      "properties": {
+        "acl": { "$ref": "#/definitions/acl" }
```

Comment:
Use `xdm` or `repo` as prefix for all properties. If you want to use `repo`, we'll have to add it to the meta-schema files.